assets/js/signup.js

```diff
@@ -90,13 +113,18 @@ var questionRules = function(question) {
     rule["required"] = true;
   }
   if (question.validationType != null) {
+    // There should really be some form of validation for the validationType
+    // One option would be to make a new table with all of the possible validation types
+    // which would be both a foreign key for the validationType field in the questionInformation table
+    // and a way to validate at this point too. If the validationType has a typo for any
+    // given field, the front-end form validation fails to run at all for any of the fields.
```

Comment:
I see. How many of these would actually be used though? And how many would be used more than once? Because, if we want functionality for all of those, then we will have to make this validationType thing support multiple for a single question (as is the case with min, max, and step -- all of those would be on a number input). 

Adding a ValidationType table (or enum) seems over-engineered to me since more likely than not, we would only use each validation type once.